frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -153,22 +152,23 @@
   "Add a parallel task": "Add a parallel task",
   "Task does not exist": "Task does not exist",
   "No Tasks": "No Tasks",
-  "Select task": "Select task",
+  "Select Task": "Select Task",
```

Comment:
Task is not a proper noun so no need to capitalize it.

Reply:
Task as well here is reference to resource for Tekton

![image](https://user-images.githubusercontent.com/5129024/100908175-37088500-34f1-11eb-945d-30013545a37b.png)

Reply:
No need to change then